OpenRA.Mods.Cnc/Traits/PortableChrono.cs

```diff
@@ -231,7 +234,7 @@ protected override IEnumerable<IRenderable> RenderAnnotations(WorldRenderer wr,
 				self.CenterPosition,
 				WDist.FromCells(self.Trait<PortableChrono>().Info.MaxDistance),
 				0,
-				Color.FromArgb(128, Color.LawnGreen),
+				Color.FromArgb(128, info.TargetLineColor),
```

Comment:
This isn't a target line! It is also being unhardcoded by #17919, so it would be best to drop it from here.